gradle/wrapper/gradle-wrapper.properties

```diff
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.6.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.1-all.zip
```

Comment:
Just do double check arctic fox is needed only when using AGP 7+ right?
This change wouldn't affect the IDE requirements.

Reply:
I believe that the customer project is using this version on AS `v4.2.2`

Reply:
Artic fox is for the AGP 7.0, which is still in RC. Gradle wrapper 7.1 is stable and works with latest stable Android Studio :)